src/Microsoft.ApplicationInsights.AspNetCore/TelemetryInitializers/AzureWebAppRoleEnvironmentTelemetryInitializer.cs

```diff
@@ -10,7 +10,7 @@
     /// <summary>
     /// A telemetry initializer that will gather Azure Web App Role Environment context information.
     /// </summary>
-    internal class AzureWebAppRoleEnvironmentTelemetryInitializer : TelemetryInitializerBase
+    public class AzureWebAppRoleEnvironmentTelemetryInitializer : TelemetryInitializerBase
```

Comment:
My comment about sealing applies more-so to those derived from `TelemetryInitializerBase`